mk/docker.mk

```diff
@@ -11,7 +11,7 @@ define build_image
 $(DOCKER_REGISTRY)/$(1):$(BUILD_INFO_VERSION)$(if $(IMAGE_ARCH_TAG_ENABLED),-${GOARCH},)
 endef
 
-export KUMA_BASE_IMAGE ?= kumahq/base-debian11:no-push
+export KUMA_BASE_IMAGE ?= kumahq/base-nossl-debian11:no-push
```

Comment:
Where is this image coming from?

Reply:
This is our internal image, just wanted to reflect that we base it on `nossl` version of `distroless`.

Reply:
Ok I got what was confusing me :) 
Should we make the base image name and arg in `kuma-dp` and `kumactl` ? I struggled fitting things together in this PR because of this :)

Reply:
> base image name and arg

do you mean **an** arg? 

> I struggled fitting things together in this PR because of this :)

this PR really does not change this, it's only a rename of the base image to reflect that our base image is now based on distroless nossl base image.

Reply:
Yes an arg and yes agree it's aside of this PR. I should have mentioned it was a suggestion